index.html

```diff
@@ -71,6 +71,14 @@
                 <i class="fa fa-volume-up fa-lg"></i>
             </div>
             <br>
+            <br>
+            <div id="zen-video-description"></div>
+            <br>
+            <button class="btn" id="toggleDescription">
+                Show Description
+            </button>
+            <br/>
```

Comment:
After we move `#toggleDescription`, I think there should only be one `<br>` on either side of `#zen-video-description`